### DISCUSSION TITLE
PetiteVue

### DISCUSSION THREAD

**Discussion by PSanilP** [vkurko/calendar#122] [Show and tell]
Made this with PetiteVue
https://codepen.io/psanilp/pen/KKGRWZo

Would like to know how to capture event drag and drop events

**Comment by vkurko** [comment]
Please check [eventDragStart](https://github.com/vkurko/calendar#eventdragstart), [eventDragStop](https://github.com/vkurko/calendar#eventdragstop) and [eventDrop](https://github.com/vkurko/calendar#eventdrop) options.
